src/ConnectedMode/Binding/BindingStrings.resx

```diff
@@ -117,11 +117,30 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CannotDownloadQualityProfileForLanguage" xml:space="preserve">
```

Comment:
I moved the required strings to this resource file and removed them from the one in `Integration`.